stmt/in.go

```diff
@@ -11,6 +11,18 @@ type In struct {
 	Value      Expression
 }
 
+// NewInValue returns an expression that is used with In instances
+func NewInValue(values ...interface{}) Expression {
+	if len(values) == 1 {
+		if enc, ok := values[0].(StatementEncoder); ok {
+			if selekt, ok := enc.Statement().(Select); ok {
```

Comment:
we can probably just return the `Statement` here without checking if it's a `Select`